extra/grafana/kubernetes/StarRocks-Overview-kubernetes-3.0.json

```diff
@@ -8481,10 +8481,10 @@
         "includeAll": false,
         "label": null,
         "multi": false,
-        "name": "be_instance",
+        "name": "instance",
```

Comment:
similar issue to this `instance`, there is a `fe_instance` and so what does this `instance` means to user.